pkg/ema.py

```diff
@@ -251,6 +252,7 @@ def setup_message(message_name, type='binary', test=False):
                 'C:/Users/Obesity_Project/Desktop/Patient-Caregiver Relationship/Patient-Caregiver-Relationship/DeploymentInformation.db')
```

Comment:
I talked to Ashley and the new Laptops have different paths to the Patient-Caregiver Relationship folder
Laptop 1: C:\Users\Andromeda\Desktop\Patient-Caregiver-Relationship
Laptop 2: C:\Users\M2FED_LAPTOP\Desktop\Patient-Caregiver-Relationship

Do you have any recommendations as to how we can access DeploymentInformation.db in more effective way?

Reply:
feel like we relative path can solve it. The difference is only the OS user name....
but i remember we cannot because we are currently a submodule within the acoustic folder
while in testing/development, we are independent....

if our recommender is separated into an independent system, we can change to relative path
for now.... the straightforward solution is to check which path exists